Joker/src/components/home-screen.title.tsx

```diff
@@ -0,0 +1,11 @@
+import {RouteProp} from '@react-navigation/native';
+import React from 'react';
+import {RootStackParamList} from '../types';
+import {Text} from 'react-native';
+
+export type HomeScreenTitleProps = {
+  route: RouteProp<RootStackParamList>;
```

Comment:
Do not use navigation props, look in the haqq-wallet useTypedRoute hook

Reply:
done